### PR TITLE
Fix issue where other heading levels are disabled when Heading 1 is selected

### PR DESCRIPTION
**Pull request by Keito654**
Fix issue #197 

I have modified the code by moving the variable` level `outside the scope of the if statement and renamed it to `currentLevel `for greater specificity. Additionally, I have included `currentLevel `in the conditions for `canSetHeading`.

Please excuse any oversights as I am new to contributing and still familiarizing myself with the process.

### CODE REVIEW

File: src/controls/MenuSelectHeading.tsx [hunk 1]
```diff
@@ -173,19 +174,21 @@ export default function MenuSelectHeading({
...
+          currentLevel as keyof typeof LEVEL_TO_HEADING_OPTION_VALUE
         ];
     }
   }
 
   const isCurrentlyParagraphOrHeading = selectedValue !== "";
   const canSetParagraph = editor?.can().setParagraph();
   // We have to pass a level when running `can`, so this is just an arbitrary one
-  const canSetHeading = editor?.can().setHeading({ level: 1 });
+  const canSetHeading =
```
Comment (sjdemartini) [inline src/controls/MenuSelectHeading.tsx:190]: Would you be able to add a comment about why we're using this logic and link to the GitHub issue?

**Review by sjdemartini — COMMENTED** [review]
Thank you for opening this! I'll aim to test this later, but generally looks good to me!

File: src/controls/MenuSelectHeading.tsx (hunk 1 above)
Comment (Keito654) [reply src/controls/MenuSelectHeading.tsx:190]: OK! I'll work on it.

Comment (Keito654) [reply src/controls/MenuSelectHeading.tsx:193]: I added a comment. Please check.

**Pull request merged**